filer/models/foldermodels.py

```diff
@@ -145,6 +149,12 @@ class Meta:
         verbose_name = _("Folder")
         verbose_name_plural = _("Folders")
 
+    def __str__(self):
+        return self.pretty_logical_path
+
+    def __repr__(self):
+        return '<{}: {}>'.format(self.__class__.__name__, self.pretty_logical_path)
```

Comment:
Should be just use f-strings everywhere?

`find filer -name "*.py" | xargs pyupgrade --py37-plus`. This can help in auto upgrading pretty easily.

Reply:
I'd agree, but was thinking that pre-commit could introduce pyupgrade and the usual helpful hooks in a separate PR.